install/build_from_source.md

```diff
@@ -62,9 +62,17 @@ To have ROOT setup automatically at each login, that command can be appended to
 
 > TODO: explain how ROOT features/STL backports interact with C++ standards and how to pick a standard
 
-## Enabling experimental features, aka ROOT 7
+## Enabling experimental features, aka ROOT7
 
-> TODO: explain how to build ROT with ROOT 7 enabled
+New and improved versions of standard ROOT components are being implemented under codename ROOT7.
+
+These features require C++ standard 14 or higher. To make sure ROOT7 features get compiled, you can specify the cmake
+option `-Droot7=ON`. This will automatically set the C++ standard with which ROOT is compiled to C++14.
```

Comment:
I'd say "turn C++14 on using `-DCMAKE_CXX_STANDARD=14`; this will automatically enable the `root7` feature"?